src/lib/components/workflow/search-attribute-input/list-input.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import { afterUpdate, onMount } from 'svelte';
+
+  import ChipInput from '$lib/holocene/input/chip-input.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import type { SearchAttributeInputValue } from '$lib/stores/search-attributes';
+
+  export let value: SearchAttributeInputValue;
+  let _value = [];
```

Comment:
I feel like `afterUpdate` and `onMount` are usually code smells—particularly if they're _not_ either waiting for the DOM nodes to load and/or cleaning up DOM side effects.